packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -245,28 +293,42 @@ contract Colosseum is Initializable, Semver {
     {
         require(_outputIndex > 0, "Colosseum: challenge for genesis output is not allowed");
 
-        Types.Challenge storage challenge = challenges[_outputIndex];
+        Types.Challenge storage challenge = challenges[_outputIndex][msg.sender];
+
+        if (challenge.turn >= TURN_INIT) {
+            ChallengeStatus status = _challengeStatus(challenge);
+            require(
+                status == ChallengeStatus.CHALLENGER_TIMEOUT,
+                "Colosseum: the challenge for given output index is already in progress"
+            );
+
+            _challengerTimeout(_outputIndex, msg.sender);
+        }
+
+        Types.CheckpointOutput memory targetOutput = L2_ORACLE.getL2Output(_outputIndex);
 
         require(
-            !_isInProgress(challenge),
-            "Colosseum: the challenge for given output index is already in progress or approved."
+            targetOutput.timestamp + CREATION_PERIOD >= block.timestamp,
+            "Colosseum: cannot create a challenge after the creation period"
```

Comment:
Could you explain me why the creation period is needed? Is this because you want to give the security council enough time to dismiss the challenge?

Reply:
yes, you're right